src/Plugin.php

```diff
@@ -70,9 +70,11 @@ public static function post_gallery($output = '', $atts) {
     $atts['post_type'] = 'attachment';
     $atts['post__in'] = $atts['include'];
     $layout = !empty($atts['layout']) ? $atts['layout'] : apply_filters('gallerya/default_layout', self::DEFAULT_LAYOUT);
+    $min_images_to_show_nav = apply_filters('gallerya/min_images_to_show_nav', 5);
```

Comment:
1. The variable + hook name has the subject and context reversed.  The subject should always come first, followed by contextual information and/or qualifiers; i.e.:

    ```diff
    -min_images_to_show_nav
    +nav_minimum_images
    +nav_min_image_count
    +nav_min_count
    +nav_minimum_count
    +nav_count_min
    ```
    The last one, `nav_count_min` seems to match the purpose most.

    It also sounds almost like an internal option name of Flickity/Slick – did you check whether there is an option in JS as well and how it's called in there?

1. Note that the previous default value was 6, not 5.  Did you change it intentionally as part of this PR/commit?